packages/adapters/wagmi/src/client.ts

```diff
@@ -252,6 +252,7 @@ export class WagmiAdapter implements ChainAdapter {
   public construct(appKit: AppKit, options: AppKitOptionsWithCaipNetworks) {
     this.appKit = appKit
     this.options = options
+    this.caipNetworks = options.networks
```

Comment:
We already do this in constructor btw

Reply:
Ahh I think for images we need to get the updated network objects from AppKit right? Got it